charts/kangal/Chart.yaml

```diff
@@ -1,12 +1,12 @@
-apiVersion: v1
+apiVersion: v2
 description: Run JMeter performance tests in Kubernetes cluster with Kangal
 keywords:
   - kubernetes
   - jmeter
   - performance tests
   - tests runner
 name: kangal
-version: 1.4.10
+version: 1.5.0
```

Comment:
I would say it should be major version bump, because of the `apiVersion: v2` and some breaking changes in templates